base/intfuncs.jl

```diff
@@ -978,7 +978,7 @@ end
 
 Return an array with element type `T` (default `Int`) of the digits of `n` in the given
 base, optionally padded with zeros to a specified size. More significant digits are at
-higher indices, such that `n == sum(digits[k]*base^(k-1) for k=1:length(digits))`.
+higher indices, such that `n == sum(digits[k]*base^(k-1) for k in eachindex(digits))`.
```

Comment:
I think this was clearer before since one is taking an exponential of `k`. The genericness of iterating indices just takes away from the point being made.

Reply:
That's a reasonable position. OTOH, with 1:length, you have to worry about bounds errors which also distracts from the point. Personally, I think it's similar either way.